applications/OptimizationApplication/tests/control/thickness/test_shell_thickness_control.py

```diff
@@ -22,6 +22,13 @@ def setUpClass(cls):
             "filter_settings": {
                 "filter_type"  : "implicit_filter",
                 "filter_radius": 0.2
+            },
+            "beta_settings": {
+                "initial_value": 25,
+                "max_value"    : 30,
+                "adaptive"     : false,
```

Comment:
please add unittest and check that betta is corerctly updated.

Reply:
done :)